src/drivers/navio_gpio/test/main.cpp

```diff
@@ -0,0 +1,6 @@
+extern int do_test();
+
+int main()
+{
+	return do_test();
+}
```

Comment:
Yes. Just like the tests for DriverFramework drivers. Although, for these, you need to uncomment add_subdirectory in CMakeLists.txt manually in the parent directories. Then you get a separate test executable.

At least for the RGBLED driver's test, I will change the CMakeLists.txt when I move it to under DriverFramework.